### DISCUSSION TITLE
Notification bar tile is invalid

### DISCUSSION THREAD

**Discussion by Menma01** [Mygod/VPNHotspot#229] [Q&A]
Hey guys,after the Android 11 ​​system reboot, the WLAN hotspot of the notification bar tile is gray and unavailable, and it needs to be connected to WIFI to recover. This is abnormal. The data hotspot switch should not be affected by the WIFI connection.
![IMG_20210525_093131.jpg](https://user-images.githubusercontent.com/84641481/119426596-20d4c800-bccf-11eb-8191-670f0242d5bf.jpg)

**Comment by Mygod** [comment]
Hey looks like you have too many custom tiles enabled. Due to AOSP problematic design, you need to wait for it to get enabled (it might take a long time).
